packages/components/src/InputTime/InputTime.tsx

```diff
@@ -52,3 +48,33 @@ export function InputTime({
     }
   }
 }
+
+function dateToTimeString(date?: Date): string {
+  if (date == undefined || !(date instanceof Date)) {
```

Comment:
I don't think you need the `date == undefined` check since `undefined instanceof Date` will return `false`

Reply:
Yeah, good call. Updated.